.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,14 +4,18 @@ about: Suggest an idea for this project
 labels: enhancement, awaiting triage
 ---
 
-## Problem Description
-<!-- Please provide a clear and concise description of what the problem or issue the proposed feature will solve (Ex. I'm always frustrated when ...) -->
+## Problem
+<!-- Describe a problem solved by this feature; or delete the section entirely. -->
```

Comment:
I'm not sure this should be optional

Reply:
I don't think all features necessarily solve problems, they are sometimes new capabilities.